main.tf

```
@@ -39,7 +39,7 @@ module "resource_group" {
 
 module "postgresql_server" {
   source  = "terraform.registry.launch.nttdata.com/module_primitive/postgresql_server/azurerm"
-  version = "~> 1.0"
+  version = "~> 1.1.0"
```

Are we sure this is what we want? Perhaps you mean something more like this? 
```suggestion
  version = "~> 1.1"
```

This will set the minimum version to 1.1, but allow for 1.2, 1.3, 1.4, ... 1.N but NOT 2.0.